tests/e2e/fixtures/replication_slot/cluster-pg-replication-slot.yaml.template

```diff
@@ -4,6 +4,7 @@ metadata:
   name: cluster-pg-replication-slot
 spec:
   instances: 3
+  imageName: "${POSTGRES_IMG}"
```

Comment:
I think this should not be required. If the default image is postgres <= 10 it should fail even if not explicitly stated. Otherwise, it's a bug.

Reply:
For manual testing this was not working as expected.

Reply:
I mean for running locally

Reply:
I would remove it anyway because it is not needed for the test.